hack/tests/e2e-helm-new-shell.sh

```diff
@@ -0,0 +1,132 @@
+
+    if ! timeout 1m kubectl rollout --namespace=${test_namespace} status deployment.apps/${nginx_deployment};
+    then
+        kubectl get events --namespace=${test_namespace}
+        kubectl describe --namespace=${test_namespace} pods -l "app.kubernetes.io/instance=${release_name}"
+        kubectl describe --namespace=${test_namespace} deployments ${nginx_deployment}
+        kubectl logs deployment.apps/nginx-operator-controller-manager -c manager
+        exit 1
+    fi
+
+    nginx_service=$(kubectl get --namespace=${test_namespace} service -l "app.kubernetes.io/instance=${release_name}" -o jsonpath="{..metadata.name}")
+    kubectl get --namespace=${test_namespace} service ${nginx_service}
+
+    # scale deployment replicas to 2 and verify the
+    # deployment automatically scales back down to 1.
+    kubectl scale --namespace=${test_namespace}  deployment.apps/${nginx_deployment} --replicas=2
+    if ! timeout 1m bash -c -- "until test \$(kubectl get --namespace=${test_namespace} deployment/${nginx_deployment} -o jsonpath='{..spec.replicas}') -eq 1; do sleep 1; done";
+    then
+        kubectl get events --namespace=${test_namespace}
+        kubectl describe --namespace=${test_namespace} pods -l "app.kubernetes.io/instance=${release_name}"
+        kubectl describe --namespace=${test_namespace} deployments ${nginx_deployment}
+        kubectl logs deployment.apps/nginx-operator-controller-manager -c manager --namespace=${test_namespace}
+        exit 1
+    fi
+
+    # update CR to replicaCount=2 and verify the deployment
+    # automatically scales up to 2 replicas.
+    kubectl patch --namespace=${test_namespace}  Nginx nginx-sample -p '[{"op":"replace","path":"/spec/replicaCount","value":2}]' --type=json
+    if ! timeout 1m bash -c -- "until test \$(kubectl get --namespace=${test_namespace} deployment/${nginx_deployment} -o jsonpath='{..spec.replicas}') -eq 2; do sleep 1; done";
+    then
+        kubectl get events --namespace=${test_namespace}
+        kubectl describe --namespace=${test_namespace} pods -l "app.kubernetes.io/instance=${release_name}"
+        kubectl describe --namespace=${test_namespace} deployments ${nginx_deployment}
+        kubectl logs deployment.apps/nginx-operator-controller-manager -c manager --namespace=${test_namespace}
+        exit 1
+    fi
+
+    kubectl delete --namespace=${test_namespace} -f config/samples/example_v1alpha1_nginx.yaml --wait=true
+    kubectl logs deployment.apps/nginx-operator-controller-manager -c manager --namespace=${test_namespace} | grep "Uninstalled release" | grep "${release_name}"
+}
+
+# create and build the operator
+
+mkdir nginx-operator
+cd nginx-operator
+log=$(operator-sdk init --plugins=helm.operator-sdk.io/v1 \
+  --domain=com --group=example --version=v1alpha1 --kind=Nginx \
+  2>&1)
+echo $log
+if echo $log | grep -q "failed to generate RBAC rules"; then
+    echo FAIL expected successful generation of RBAC rules
+    exit 1
+fi
+
+pushd ../nginx-operator
```

Comment:
What's the point of this? Based on line 106, it looks like we're already in the nginx-operator directory.